build.rs

```diff
@@ -80,6 +80,22 @@ fn build_libipt() {
     env::set_current_dir(&Path::new("..")).unwrap();
 }
 
+fn fetch_libipt() {
+    eprintln!("Fetch libipt...");
```

Comment:
Shouldn't this be "Build libipt" (and do we need the `...`?).

Reply:
There are separate build and fetch steps as we need to build a couple of files inside the libipt repo regardless of whether the user wants to use their own libipt or ours.

There's a comment later that tries to explain, I can move it up if you think it would be clearer.

Reply:
`...` could go, yeah.

Reply:
Ah, yes, I see now. Moving the comment might help.